CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 2.11.11
+- Ensure compatibility with Java 11 (see PR #775)
+- [spring-xsuaa-starter] ``xsuaaMtlsRestOperations`` and ``xsuaaRestOperations`` are only auto-configured if ``XsuaaServiceConfiguration`` bean is given
+
+#### Dependency upgrades
```

Comment:
I think here are some dependency upgrades missing

Reply:
i've only added those that gets shipped / packaged